server/utils/tasks/listTable.ts

```diff
@@ -28,6 +29,32 @@ const getTaskType = (task: Task): string => {
   return taskType
 }
 
+const getDecisionOutcome = (outcome: string): string => {
+  switch (outcome) {
+    case 'accepted':
+      return 'Accepted'
+    case 'rejected':
+      return 'Rejected'
+    case 'withdraw':
+      return 'Withdraw'
+    case 'withdrawn_by_pp':
+      return 'Withdrawn by pp'
+    default:
+      return ''
+  }
+}
+
+export const getDecision = (task: Task): string => {
```

Comment:
Looks like we're missing a case for `PlacementRequestTask`s?

Reply:
but we are requesting only 'PlacementApplication', 'Assessment'

Reply:
I think it'd be best to have it there for completeness, as this part of the code doesn't have any knowledge of what we've requested at the controller level. If this changes for any reason, or we try and use this code in a place where we request all types, we could end up with unintended consequences.